package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "test": "standard && tap test/*.test.js",
-    "mysql": "docker run -d -p 3306:3306 -e MYSQL_ALLOW_EMPTY_PASSWORD=yes --rm mysql"
+    "mariadb": "docker run -d -p 3306:3306 -e MYSQL_ALLOW_EMPTY_PASSWORD=yes --rm mariadb",
```

Comment:
could we put a version also for mariadb? WDYT?

Reply:
It should work on all versions (including the latest) but we can set it to 10.1 tag as it is the version that is shipped with Debian ^^